src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1712,12 +1717,14 @@ a.tooltip:hover::after {
 	content: "\00a0";
 }
 
-/* This animation is layered onto the mistake-proofing delay for dismissing
-	a hovered tooltip, to ensure it feels responsive even with the delay.
-	*/
-.fade-out {
-	opacity: 0;
-	transition: opacity 0.45s cubic-bezier(0, 0, 0.1, 1.0);
+@media not (prefers-reduced-motion) {
```

Comment:
Could you merge both of them? I'd prefer if we had as little media queries as possible.